tests/travis-script.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+../../occ face:setup
+make test
```

Comment:
It does the tests, and they fail because the files don't exist anymore. :confused: :confounded: :sob: 

@stalker314314 Do you have any idea?

Reply:
no. Which files? I think you are too deep in this at this point:)

Reply:
:sweat_smile: 
Add comments to the commit to explain a bit, but don't focus on it.

See travis.. it is easier to see the problem ..

Well, the model is installed..
 * https://travis-ci.org/matiasdelellis/facerecognition/jobs/620595745#L1253

..but when trying to use them he doesn't find them ..
 * https://travis-ci.org/matiasdelellis/facerecognition/jobs/620595745#L1342

Reply:
this is weirdest thing I am seeing recently:) I have no idea...can you try doing `ls -alR` sprinkled all in `make test` Makefile, so we see if when it is missing? At beginning, at end and in between all of these `test-bin-deps vendor-deps js-templates` targets

Reply:
Model folder is empty...
 * https://travis-ci.org/matiasdelellis/facerecognition/jobs/620659199#L1288

Install models... 
 * https://travis-ci.org/matiasdelellis/facerecognition/jobs/620659199#L1293

Model folder now has the models..
 * https://travis-ci.org/matiasdelellis/facerecognition/jobs/620659199#L1350

Build the app (Since here don't install the models anymore maybe is not needed...): This should not affect appdata..
 * https://travis-ci.org/matiasdelellis/facerecognition/jobs/620659199#L1357

Run tests ..
 * https://travis-ci.org/matiasdelellis/facerecognition/jobs/620659199#L1420

It fails because it does not exist .. :disappointed: 
 * https://travis-ci.org/matiasdelellis/facerecognition/jobs/620659199#L1430

Appdata is empty.. :confused: 
https://travis-ci.org/matiasdelellis/facerecognition/jobs/620659199#L1467

Reply:
https://github.com/nextcloud/travis_ci/blob/master/core_install.sh#L108

:disappointed:

Reply:
omfg, lol
So...tests need to install models?

Reply:
> ~https://github.com/nextcloud/travis_ci/blob/master/core_install.sh#L108~

I'm still wrong.. This command is executed before..